datacite-findable-doi-handler/src/test/java/no/unit/nva/datacite/handlers/FindableDoiEventHandlerTest.java

```diff
@@ -131,19 +128,21 @@ public void handleRequestThrowsIllegalArgumentExceptionOnMissingPublicationId()
         assertThat(exception.getMessage(), containsString(PUBLICATION_ID_FIELD_INFO));
     }
 
-    @Test
-    public void handleRequestThrowsIllegalArgumentExceptionOnInvalidDoi() {
-        InputStream inputStream = IoUtils.inputStreamFromResources(
-            "doi_publication_event_invalid_doi.json");
-        IllegalArgumentException exception = assertThrows(IllegalArgumentException.class,
-                                                          () -> findableDoiHandler.handleRequest(inputStream,
-                                                                                                 outputStream,
-                                                                                                 context));
+    //TODO: Is it necessary to validate the DOI before sending to Datacite? If Datacite says it is OK, do we need to
```

Comment:
Not sure if we should delete this test. My opinion is that we should delete it. As it is fairly impossible to make a non valid doi findable.

Reply:
It is impossible